IBPSA/Resources/Scripts/Dymola/ConvertIBPSA_from_4.0.0_to_4.0.1.mos

```diff
@@ -0,0 +1,790 @@
+// Conversion script for IBPSA library
+//  from: Version 4.0.0
+//  to:   Version 4.0.1
+clear
```

Comment:
Because the current version of the `IBPSA` library is 3.0.0, you should call the new version `4.0.0`, and put your conversions to the existing file `ConvertIBPSA_from_3.0_to_4.0.mos`